chapters/1.md

```diff
@@ -3,37 +3,63 @@
+
+To do that, banks and individuals can use something called blockchain. Blockchain is a technology that is a distributed database that continuously adds to a record of all the transactions that have occurred within a network. This database is often referred to as a ledger, because the data contains a large list of transactions that have taken place.
+
+The first distributed blockchain was conceptualized in 2008 by an anonymous person or group known as Satoshi Nakamoto and implemented in 2009 as a core component of Bitcoin, where it serves as the public ledger for all transactions on the Bitcoin network. The invention of the blockchain made Bitcoin the first digital currency to solve the double spending problem without the need for a trusted authority or central server. 
+
+Within a distributed network, if you have many disparate parties with no central authority, how do you ensure they all agree? The blockchain is a way of maintaining consensus in a decentralized distributed network. Transactions are grouped into “blocks”, which are linked and secured using cryptography, and every time a new block is solved (i.e. checked, signed, and verified by network participants), these transactions are recorded on the blockchain in chronological order. What goes into a block is just data, but it can be anything. The most obvious example is financial transactions.
+
+Consensus within the network is maintained because every time a new block is solved (usually every 10 minutes or so), all participants in the network must agree on the state of the blockchain; When this consensus occurs, each network participant's copy of the blockchain syncs so they all match. That way all copies march in lockstep, even though there is no one central master copy. As a result, the blockchain is completely unalterable as a historical record. If you take your copy of the blockchain and go back and change something in an earlier block, your copy of the blockchain will simply be rejected in 10 minutes at the next re-sync because your copy would not match the rest. This provides security to the network and removes the need for a centralized trusted third party. So, cryptocurrency can conceivably work as the central bank in the real-time gross settlement system we had discussed before, without needing to trust that central bank, or your bank, or any third party.
```

Comment:
-- (usually every 10 minutes or so)
++ (creating blocks time depends on the cryptocurrency you'll use)

Reply:
This isn't my text, but possibly was marked as added in the diff because of how much text I added at the beginning. I hadn't really attempted to do a rewrite on any of the blockchain or cryptocurrency text.